<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSDeviceTemperaturePanel</class>
 <widget class="QWidget" name="CSDeviceTemperaturePanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBoxArm">
     <property name="title">
      <string>ARM Chip Temperature（-20℃~60℃）</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="8,1">
      <property name="spacing">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>18</number>
      </property>
      <property name="bottomMargin">
       <number>18</number>
      </property>
      <item>
       <widget class="QLineEdit" name="lineEditArm">
        <property name="text">
         <string>-- -- --</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelArm">
        <property name="text">
         <string>℃</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxMainBoard">
     <property name="title">
      <string>Main Board Temperature（-20℃~80℃）</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="8,1">
      <property name="spacing">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>18</number>
      </property>
      <property name="bottomMargin">
       <number>18</number>
      </property>
      <item>
       <widget class="QLineEdit" name="lineEditMainBoard">
        <property name="text">
         <string>-- -- --</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelMainBoard">
        <property name="text">
         <string>℃</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxSlaveBoard">
     <property name="title">
      <string>Slave Board Temperature（-20℃~80℃）</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="8,1">
      <property name="spacing">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>18</number>
      </property>
      <property name="bottomMargin">
       <number>18</number>
      </property>
      <item>
       <widget class="QLineEdit" name="lineEditSlaveBoard">
        <property name="text">
         <string>-- -- --</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelSlaveBoard">
        <property name="text">
         <string>℃</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxFPGA">
     <property name="title">
      <string>FPGA Temperature（-20℃~80℃）</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="8,1">
      <property name="spacing">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>18</number>
      </property>
      <property name="bottomMargin">
       <number>18</number>
      </property>
      <item>
       <widget class="QLineEdit" name="lineEditFPGA">
        <property name="text">
         <string>-- -- --</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelFPGA">
        <property name="text">
         <string>℃</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxCMOS">
     <property name="title">
      <string>CMOS Temperature（-20℃~80℃）</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="8,1">
      <property name="spacing">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>18</number>
      </property>
      <property name="bottomMargin">
       <number>18</number>
      </property>
      <item>
       <widget class="QLineEdit" name="lineEditCMOS">
        <property name="text">
         <string>-- -- --</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelCMOS">
        <property name="text">
         <string>℃</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxChamber">
     <property name="title">
      <string>Temperature Chamber（10℃~30℃）</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="8,1">
      <property name="spacing">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>18</number>
      </property>
      <property name="bottomMargin">
       <number>18</number>
      </property>
      <item>
       <widget class="QLineEdit" name="lineEditChamber">
        <property name="text">
         <string>-- -- --</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelChamber">
        <property name="text">
         <string>℃</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
